controllers/upgrade.js

```diff
@@ -0,0 +1,45 @@
+const PouchDB = require('pouchdb-core');
+PouchDB.plugin(require('pouchdb-adapter-http'));
+
+const buildDbUrl = 'https://staging.dev.medicmobile.org/_couch/builds';
+const buildDb = new PouchDB(buildDbUrl);
+const targetDbUrl = process.env.COUCH_URL;
+const targetDb = new PouchDB(targetDbUrl);
+
+module.exports = (version, username) => {
+  console.log('upgrade()', `Upgrading to ${version}…`);
+
+  console.log('upgrade()', 'Fetching newDdoc…');
+  return buildDb
+    .get(version, { attachments:true })
```

Comment:
If you use just the version as the ID then you can only publish one product to each market. Is this on purpose? Marc and I have talked about publishing medic-reporter to horti too so we can get rid of dashboard from the instances. This would require a separate builds db. Can we instead use "[product-name]:[version]" as the key?